src/Sdk/WebApi/WebApi/BrokerHttpClient.cs

```diff
@@ -111,5 +117,33 @@ public class BrokerHttpClient : RawHttpClientBase
 
             throw new Exception($"Failed to get job message: {result.Error}");
         }
+
+        public async Task<TaskAgentSession> CreateSessionAsync(
+
+           TaskAgentSession session,
+           CancellationToken cancellationToken = default)
+        {
+
+            var requestUri = new Uri(Client.BaseAddress, "session");
+            var requestContent = new ObjectContent<TaskAgentSession>(session, new VssJsonMediaTypeFormatter(true));
+
+            var result = await SendAsync<TaskAgentSession>(
+                new HttpMethod("POST"),
+                requestUri: requestUri,
+                content: requestContent,
+                cancellationToken: cancellationToken);
+
+            if (result.IsSuccess)
+            {
+                return result.Value;
+            }
```

Comment:
Yeah deep in the chain it uses:
```
//
        // Summary:
        //     Gets a value that indicates if the HTTP response was successful.
        //
        // Returns:
        //     true if System.Net.Http.HttpResponseMessage.StatusCode was in the range 200-299;
        //     otherwise, false.
        public bool IsSuccessStatusCode { get; }
 ```

So 201 should be fine